mmf/models/vilbert.py

```diff
@@ -53,10 +56,10 @@ def transpose_for_scores(self, x):
             self.num_attention_heads,
             self.attention_head_size,
         )
-        x = x.view(*new_x_shape)
+        x = torch.reshape(x, new_x_shape)
```

Comment:
```suggestion
        x = x.view(new_x_shape)
```

Using view is better as reshaping might make a copy of the tensor in some cases.